_layouts/kmap.html

```diff
@@ -2,110 +2,136 @@
 
 <html lang="en">
 {% include head.html %}
+<script src="../assets/js/module.js"></script>
+<script src="../assets/js/vendor/jquery-3.4.1.min.js"></script>
+<script src="../assets/js/shareable-links.js"></script>
 
```

Comment:
https://github.com/CircuitVerse/Interactive-Book/blob/master/_layouts/default.html  

do we need to include all these? they're already in the default layout

Reply:
kmap.html is an independent layout. So, it doesn't use the default.html scope variables. I've tried without using them, but jquery doesn't seem to work. Hence, added.

Reply:
Without these, the dark mode won't work.

Reply:
cool, just noticed it.